src/api/docker.ts

```diff
@@ -196,7 +196,8 @@ export function killContainer(id: string): Promise<void> {
           if (containerInfo.State.Status === 'exited') {
             return container.remove();
           } else if (containerInfo.State.Status === 'running') {
-            return container.kill();
+            return container.stop()
```

Comment:
I think you need another checking if Status is 'stopped' and only removing in that case.
we had like this before and didn't worked good.